index.html

```diff
@@ -16,6 +16,18 @@ <h1>Rev My $</h1>
     </div>
     
     <h3>Hello, yourName</h3>
+
+     <!--monthly income input  -->
+     <div class="goalsAndExpenses">
```

Comment:
Is there a .goalsAndExpenses class in the css? If not I would delete this class for now.

Reply:
The class has been created in HTML to be referenced in CSS later on.